.circleci/local_publish_helpers.sh

```diff
@@ -5,11 +5,12 @@ default_verdaccio_package=verdaccio@5.1.2
 
 function startLocalRegistry {
     # Start local registry
-    tmp_registry_log=$(mktemp)
+    tmp_registry_log="$(mktemp)"
     echo "Registry output file: $tmp_registry_log"
     (cd && nohup npx ${VERDACCIO_PACKAGE:-$default_verdaccio_package} -c $1 &>$tmp_registry_log &)
     # Wait for Verdaccio to boot
-    grep -q 'http address' <(tail -f $tmp_registry_log)
+    sleep 60
+    # grep -q 'http address' <(tail -f $tmp_registry_log)
```

Comment:
why hardcoded sleep instead of waiting for log statement?

Reply:
Can we do a curl or other health check to wait until verdaccio is online?

Reply:
Syntax for "<(tail -f $tmp_registry_log)" was not compatible with this POSIX implementation/codebuild bash; but sleep 60 works fine with both CircleCI & CodeBuild

Reply:
Can we find other solution that doesn't involve fixed sleeping ?

Reply:
adding this to [backlog](https://app.asana.com/0/1203848575872011/1204779956749776/f)